manifests/module.pp

```diff
@@ -44,11 +53,11 @@
   validate_absolute_path($::selinux::config::module_build_dir)
   validate_absolute_path($::selinux::refpolicy_makefile)
 
-  $module_dir = "${::selinux::config::module_build_dir}/${title}"
+  $module_dir = $::selinux::config::module_build_dir
   $module_file = "${module_dir}/${title}"
 
```

Comment:
`$::selinux::config::module_build_dir` should be `$::selinux::module_build_dir`?